tools/SourceKit/tools/complete-test/complete-test.cpp

```diff
@@ -21,14 +21,26 @@
 #include "llvm/Support/FileSystem.h"
 #include <fstream>
 #include <regex>
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
```

Comment:
Why this complicated check necessary ? I'd expect that checking that `_WIN32` is defined versus not would be sufficient.